src/main/java/org/rocstreaming/roctoolkit/Check.java

```diff
@@ -25,8 +27,8 @@ static int notNegative(int value, String name) {
         return value;
     }
 
-    static long notNegative(long value, String name) {
-        if (value < 0) {
+    static Duration notNegative(Duration value, String name) {
```

Comment:
value could be null here. I guess need to rename it to `notNegativeOrNull` and check if value is null